src/globus_sdk/exc/err_info.py

```diff
@@ -64,9 +67,18 @@ def __init__(self, error_data: t.Dict[str, t.Any]):
         self.session_required_single_domain = t.cast(
             t.Optional[t.List[str]], data.get("session_required_single_domain")
         )
-        self.session_required_policies = t.cast(
-            t.Optional[str], data.get("session_required_policies")
-        )
+
+        # get str|None and parse as appropriate
+        self.session_required_policies: t.Optional[t.List[str]] = None
+        session_required_policies = data.get("session_required_policies")
+        if isinstance(session_required_policies, str):
+            self.session_required_policies = session_required_policies.split(",")
+        elif session_required_policies is not None:
+            log.warning(
+                "during ErrorInfo parsing, got unexpected type for "
```

Comment:
What are your thoughts on this phrasing?

```suggestion
                "During ErrorInfo instantiation, got unexpected type for "
```

Reply:
I'm not sure I'm convinced that there's a difference between these, but I don't have any particular preference for the prior version. I'm taking the suggestion!